{"name": "test_basic_02", "status": "broken", "statusDetails": {"message": "ValueError: dictionary update sequence element #0 has length 1; 2 is required", "trace": "self = <test_basic.TestBasic object at 0x106de7a20>, action = 'debug'\n\n    @allure.severity('blocker')\n    @allure.story('Basic')\n    def test_basic_02(self, action):\n        \"\"\"\n            用例描述：profile接口\n        \"\"\"\n        conf = Config()\n    \n        data = Basic()\n        test = Assert.Assertions()\n>       request = Send_Request.Request()\n\nTestCase/test_basic.py:55: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Common.Send_Request.Request object at 0x106de7898>\n\n    def __init__(self):\n        \"\"\"\n        :param env:\n        \"\"\"\n        conf = Config()\n        data = Basic()\n        host = conf.host_debug\n        url = conf.tokenHost_debug\n        data = conf.loginInfo_debug\n        self.headers = conf.headers\n        print(url)\n>       print(dict(data))\nE       ValueError: dictionary update sequence element #0 has length 1; 2 is required\n\nCommon/Send_Request.py:34: ValueError"}, "description": "\n            用例描述：profile接口\n        ", "start": 1586920285889, "stop": 1586920285890, "uuid": "64f84d32-8584-4dd7-81dd-eaa7014c61b3", "historyId": "00cf352422e4e739b9079d72fc93ad96", "testCaseId": "1bc9621ed29be39d8aa389abbd6af750", "fullName": "TestCase.test_basic.TestBasic#test_basic_02", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "Basic"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_basic"}, {"name": "subSuite", "value": "TestBasic"}, {"name": "host", "value": "Lettos-iMac.local"}, {"name": "thread", "value": "59904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_basic"}]}